tests/test-parse-multiline.js

```diff
@@ -23,9 +23,9 @@ t.equal(parsed.DOUBLE_QUOTES_SPACED, '    double quotes    ', 'respects surround
 
 t.equal(parsed.EXPAND_NEWLINES, 'expand\nnew\nlines', 'expands newlines but only if double quoted')
 
-t.equal(parsed.DONT_EXPAND_UNQUOTED, 'dontexpand\\nnewlines', 'expands newlines but only if double quoted')
```

Comment:
i have mixed feelings on this behavior. looks like the rubygem has done this since inception, but it doesn't follow the principle of least surprise to me.